### PR TITLE
Feat(#33)set up static questions

### PR DESCRIPTION
**Pull request by mrproper1337**
## PR Checklist

PR meets the following requirements:

- [x] Changes follow our [guidelines](https://github.com/extreme-startup/questionator/wiki/Contributing)
- [x] Tests for the changes have been added
- [ ] Docs have been added / updated

## PR Type

PR introduces the following changes:

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Other... Please describe:
```

## Actual/New behavior

Issue Number: N/A

### CODE REVIEW

**Comment by mrproper1337** [comment]
waiting on https://github.com/extreme-startup/questionator/pull/89 to be merged and aligned on top of it

File: server/package.json [hunk 1]
```diff
@@ -75,7 +78,6 @@
       "^.+\\.(t|j)s$": "ts-jest"
     },
     "coverageDirectory": "../coverage",
-    "collectCoverage": true,
```
Comment (mrproper1337) [inline server/package.json]: https://github.com/facebook/jest/issues/5739

File: server/src/modules/Question/question.controller.ts [hunk 2]
```diff
@@ -42,4 +43,9 @@ export class QuestionController {
     async delete(@Param('id') id: string) {
         return await this.questionService.delete(id);
     }
+
+    @Post('reply/:askedQuestionId')
+    reply(@Body() answer: string, @Param() { askedQuestionId }): Promise<AnswerDto> {
```
Comment (kovnick) [inline server/src/modules/Question/question.controller.ts]: Do we need this endpoint? I thought we communicate with clients through WebSockets.
Probably ask/reply methods in service are enough. Please check this with the team who developed AskedQuestions.

File: server/src/entity/AskedQuestion.ts [hunk 3]
```diff
@@ -21,16 +21,16 @@ export class AskedQuestion {
   questionId: string;
 
   @Column('varchar', {
-    name: 'generated_question',
+    name: 'question',
```
Comment (kovnick) [inline server/src/entity/AskedQuestion.ts]: Do we need migration for these changes?

File: server/src/modules/Question/question.controller.ts (hunk 2 above)
Comment (mrproper1337) [reply server/src/modules/Question/question.controller.ts]: Seems reasonable, I'll remove that endpoint and let them decide how to use service on their own

File: server/src/entity/AskedQuestion.ts (hunk 3 above)
Comment (mrproper1337) [reply server/src/entity/AskedQuestion.ts]: Done

File: server/src/migration/1555336867562-AskedQuestion.ts [hunk 4]
```diff
@@ -0,0 +1,13 @@
+import {MigrationInterface, QueryRunner} from 'typeorm';
+
+export class AskedQuestion1555336867562 implements MigrationInterface {
+
+    public async up(queryRunner: QueryRunner): Promise<any> {
+        await queryRunner.query(`ALTER TABLE "asked_question" ALTER COLUMN "generated_question" RENAME TO "question"`);
+    }
+
+    public async down(queryRunner: QueryRunner): Promise<any> {
+        await queryRunner.query(`ALTER TABLE "asked_question" ALTER COLUMN "generated_answer" RENAME TO "answer"`);
```
Comment (kovnick) [inline server/src/migration/1555336867562-AskedQuestion.ts]: should be in up() method
add to down() backward renaming:
`
await queryRunner.query(`ALTER TABLE "asked_question" ALTER COLUMN "answer" RENAME TO "generated_answer"`);
await queryRunner.query(`ALTER TABLE "asked_question" ALTER COLUMN "question" RENAME TO "generated_question"`);
`

**Comment by kovnick** [comment]
@MelnychenkoIvan Please confirm that question.service.ts with ask() and reply() methods ok for your team functionality
Ask method will change when we add support for dynamic questions

**Pull request merged**